packages/number-field/src/number-field.css

```diff
@@ -12,12 +12,11 @@ governing permissions and limitations under the License.
 
 @import './spectrum-number-field.css';
 
-:host([quiet]) {
-    width: var(--spectrum-stepper-quiet-width);
+:host {
+    inline-size: var(--mod-stepper-width, var(--spectrum-stepper-width));
 }
 
-#textfield,
-:host([quiet]) #textfield {
+#textfield {
     width: 100%;
```

Comment:
We should use logical-properties like inline-size instead of width

Reply:
Great catch!